### PR TITLE
[JA] Translate nerves 1.1.2

### PR DESCRIPTION
**Pull request by TORIFUKUKaiou**
Hi!, it's nice to meet you.

I translated ja/lessons/specifics/nerves.md section.
Please review! @elixirschool/japanese

### CODE REVIEW

File: ja/lessons/specifics/nerves.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 redirect_from:
   - /ja/lessons/advanced/nerves
 ---
 
+{% include toc.html %}
+
 ## はじめに
 
-このレッスンではNervesについて話します。Nervesプロジェクトは組込みソフトウェア開発にElixirを使用するためのフレームワークです。Nervesのウェブサイトにあるように、それはあなたが「Elixirで、安全な組み込みソフトウェアを作り、そして展開する」ことを可能にします。このレッスンは他のElixir Schoolのレッスンとは少し異なります。上級者向けのシステム設定と追加のハードウェアの両方が必要なため、Nervesを習得するのは少し困難です。したがって、初心者にはオススメしません。
+このレッスンではNervesについて話します。Nervesプロジェクトは組込みソフトウェア開発にElixirを使用するためのフレームワークです。Nervesのウェブサイトにあるように、それはあなたが「Elixirで、安全な組み込みソフトウェアを作り、そして展開する」ことを可能にします。このレッスンは他のElixir Schoolのレッスンとは少し異なります。上級者向けのシステム設定と追加のハードウェアの両方が必要なため、Nervesを習得するのは少し困難です。したがって、初心者には難しいかもしません。
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:12]:
```suggestion
このレッスンではNervesについて話します。Nervesプロジェクトは組込みソフトウェア開発にElixirを使用するためのフレームワークです。Nervesのウェブサイトにあるように、それはあなたが「Elixirで、安全な組み込みソフトウェアを作り、そして展開する」ことを可能にします。このレッスンは他のElixir Schoolのレッスンとは少し異なります。上級者向けのシステム設定と追加のハードウェアの両方が必要になるため、Nervesを習得するのは初心者には難しいかもしません。
```

File: ja/lessons/specifics/nerves.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 
-Nervesを使ってコードを書くためには、ネットワークによってこのデバイスにアクセスするための有線ネットワーク接続として、 [サポート対象のハードウェア](https://hexdocs.pm/nerves/targets.html) のいずれか、お好みのハードウェアでサポートされているメモリカードを持ったカードリーダーが必要です。
+Nervesを使ってコードを書くためには、 [サポート対象のハードウェア](https://hexdocs.pm/nerves/targets.html) のいずれかと、お好みのハードウェアでサポートされているメモリのメモリカードリーダーと、同様にネットワーク越しにデバイスにアクセスするための有線ネットワーク接続が必要です。
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:14]:
```suggestion
Nervesを使って組み込み向けのコードを書くためには、 [サポート対象のハードウェア](https://hexdocs.pm/nerves/targets.html) のいずれかと、お好みのハードウェアでサポートされているメモリカードのカードリーダーと、ネットワーク越しにデバイスにアクセスするための有線ネットワーク接続が必要です。
```

File: ja/lessons/specifics/nerves.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
 ただし、Raspberry Piには制御可能なLEDが搭載されているため、Raspberry Piを使用することをお勧めします。IExを使用したデバッグが簡単になるため、画面をターゲットデバイスに接続することをお勧めします。
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:16]:
```suggestion
ただし、Raspberry Piには制御可能なLEDが搭載されているため、Raspberry Piを使用することをお勧めします。IExを使用したデバッグが簡単になるため、モニタをターゲットデバイスに接続することをお勧めします。
```

Comment (takasehideki) [reply ja/lessons/specifics/nerves.md:16]: Original English article uses "screen". I think "モニタ" is suitable for Japanese.

File: ja/lessons/specifics/nerves.md [hunk 4]
```diff
@@ -110,24 +112,34 @@ Elapsed time: 8.022 s
 
 ## ネットワークの設定
 
-次のステップはネットワークを設定することです。Nervesのエコシステムはさまざまなパッケージを提供しています。[nerves_network](https://github.com/nerves-project/nerves_network) は、有線イーサネットポートを介してデバイスをネットワークに接続するために必要なものです。
+次のステップはネットワークを設定することです。Nervesのエコシステムはさまざまなパッケージを提供しています。[vintage_net](https://github.com/nerves-networking/vintage_net) は、有線イーサネットポートを介してデバイスをネットワークに接続するために必要なものです。
 
-あなたのプロジェクトにはすでに `nerves_init_gadget` が依存関係として存在しています。しかし、デフォルトではDHCPを使います（ `config/config.exs` の中の `config :nerves_init_gadget` の後の実行内容を確認してください）。静的IPアドレスを持つ方が簡単です。
+あなたのプロジェクトにはすでに [`nerves_pack`](https://github.com/nerves-project/nerves_pack) が依存関係として存在しています。しかし、デフォルトではDHCPを使います（ config/target.exs の中の config :vintage_net の後の設定内容を確認してください）。静的IPアドレスを持つ方が簡単です。
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:117]:
```suggestion
あなたのプロジェクトにはすでに [`nerves_pack`](https://github.com/nerves-project/nerves_pack) が依存関係として存在しています。しかし、デフォルトではDHCPを使います（ `config/target.exs` の中の `config :vintage_net` の後の設定内容を確認してください）。静的IPアドレスを持つ方が簡単です。
```

File: ja/lessons/specifics/nerves.md [hunk 5]
```diff
@@ -110,24 +112,34 @@ Elapsed time: 8.022 s
...
+         gateway: "192.168.88.1",
+         name_servers: ["8.8.8.8", "8.8.4.4"]
+       }
+     }},
+    {"wlan0", %{type: VintageNetWiFi}}
   ]
 ```
 
-この設定は有線接続用です。無線接続を使用したい場合は、[Nerves network documentation](https://github.com/nerves-project/nerves_network#wifi-networking) を参照してください。
+この設定は有線接続用です。Wi-Fiを使用したい場合は、 [VintageNet Cookbook](https://hexdocs.pm/vintage_net/cookbook.html#wifi) を参照してください。
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:142]:
```suggestion
この設定は有線接続の設定のみを更新します。WiFiを使用したい場合は、 [VintageNet Cookbook](https://hexdocs.pm/vintage_net/cookbook.html#wifi) を参照してください。
```

Comment (takasehideki) [reply ja/lessons/specifics/nerves.md:142]: I think "WiFi" is better than "Wi-Fi" as it is used in the Nerves documentation.

File: ja/lessons/specifics/nerves.md [hunk 6]
```diff
@@ -144,15 +156,15 @@ Request timeout for icmp_seq 207
 
 この出力は、デバイスがネットワークから到達可能になったことを意味します。
 
-## ネットワークファームウェアの書き込み
+## ネットワーク経由のファームウェアの書き込み
 
-これまでのところ、私達はSDカードに書き込んで、物理的にそれらを私達のハードウェアにロードしてきました。これは最初は問題ありませんが、ネットワーク経由で更新する方が簡単です。 `nerves_firmware_ssh` パッケージはまさにそれをしてくれます。デフォルトでは、すでにプロジェクトに存在し、ディレクトリ内のSSHキーを自動検出して見つけるように設定されています。
+これまでのところ、私達はSDカードに書き込んで、物理的にそれらを私達のハードウェアにロードしてきました。これは最初は問題ありませんが、ネットワーク経由で更新する方が簡単です。 [`ssh_subsystem_fwup`](https://github.com/nerves-project/ssh_subsystem_fwup) パッケージはまさにそれをしてくれます。デフォルトでは、すでにプロジェクトに存在し、 `~/.ssh` ディレクトリ内のSSHキーを自動検出して見つけるように設定されています。
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:161]:
```suggestion
これまでのところ、私達はSDカードに書き込んで、物理的にそれらを私達のハードウェアにロードしてきました。これは最初は問題ありませんが、ネットワーク経由で更新する方が簡単です。 [`ssh_subsystem_fwup`](https://github.com/nerves-project/ssh_subsystem_fwup) パッケージはまさにそれをしてくれます。プロジェクトにはデフォルトですでに設定されていて、 `~/.ssh` ディレクトリ内のSSHキーを自動検出して見つけるように構成されています。
```

File: ja/lessons/specifics/nerves.md [hunk 7]
```diff
@@ -257,6 +269,6 @@ end
 
 そして最後のステップ - アプリケーションのスーパーバイザーツリーに `{Plug.Cowboy, scheme: :http, plug: NetworkLed.Http, options: [port: 80]}` を追加します。
 
-ファームウェアのアップデート後に試すことができます。 `http://192.168.88.2/` はプレーンテキストの応答を返しており、 `http://192.168.88.2/disable` と一緒に `http://192.168.88.2/enable` はそのLEDを無効にして有効にします！
+ファームウェアのアップデート後に試すことができます。 `http://192.168.88.2/` はプレーンテキストの応答を返しており、 `http://192.168.88.2/disable` でLEDが消え、 `http://192.168.88.2/enable` でLEDが点きます！
```
Comment (takasehideki) [inline ja/lessons/specifics/nerves.md:272]:
```suggestion
ファームウェアのアップデート後に試すことができます！ `http://192.168.88.2/` はプレーンテキストの応答を返しており、 `http://192.168.88.2/disable` でLEDが消灯し、 `http://192.168.88.2/enable` でLEDが点灯します！
```

**Review by takasehideki — COMMENTED** [review]
awesome thanks @TORIFUKUKaiou !!
They are just my comment. Please feel free to refer to them.

**Comment by takasehideki** [comment]
Sorry I'm not familiar with manners/rules to maintain this repository. 
@starfish719 @marocchino Could you give an advice for us? Do you know how to do?

**Review by takasehideki — APPROVED** [review]
LGTM!!

**Review by marocchino — APPROVED** [review]

**Comment by marocchino** [comment]
sorry for the late response. happy new year! >_<

**Comment by takasehideki** [comment]
@marocchino thank you so much for merging! and a happy new year 🎉

**Comment by marocchino** [comment]
> Could you give an advice for us? Do you know how to do?

The rules of this repository are not that strict. If you are comfortable with Japanese comment, maybe you can use it.
The owners of this repository value contributors' efforts more than strict rules.

**Comment by takasehideki** [comment]
@marocchino thank you for the information. This rule is comfortable and favor for me. 
明けましておめでとうございます！ 🇯🇵

**Pull request merged**